build/run

```diff
@@ -120,14 +130,19 @@ if [ -z "${DISABLE_NESTED_DOCKER}" ]; then
     -v ${HOME}/.docker/config.json:${BUILDER_HOME}/.docker/config.json \
     -v /var/run/docker.sock:/var/run/docker.sock"
 fi
+DOCKER_API=''
+
+if (( $(bc <<< "$DOCKER_SERVER_API < $DOCKER_CLIENT_API") )); then
+    DOCKER_API="-e DOCKER_API_VERSION=${DOCKER_SERVER_API}"
+fi
```

Comment:
docker server is backward compatible with older clients, so I'm not sure that case is important, right? the case you're fixing here is newer client and older server.